nexus/test/inventory/group_vars/seed-hosts.yml

```diff
@@ -0,0 +1,62 @@
+namespace: nexus
+app_name: nexus
+
+openshift_templates_raw: "https://raw.githubusercontent.com/redhat-cop/openshift-templates"
+openshift_templates_raw_version_tag: "v1.4.12"
+
+deploy:
+  NAME: "{{ app_name }}"
+
+build:
+  NAME: "{{ app_name }}"
+  SOURCE_REPOSITORY_URL: https://github.com/redhat-cop/containers-quickstarts.git
+  SOURCE_REPOSITORY_REF: master
+  SOURCE_CONTEXT_DIR: "{{ app_name }}"
+
+nexus:
+  secret:
+    SECRET_NAME: "nexus"
+    USERNAME: admin
+    PASSWORD: "thisismytestpassword"
+  deploy:
+    VOLUME_CAPACITY: 10Gi
+    MEMORY_LIMIT: 2Gi
+    CONTAINER_IMAGE: sonatype/nexus3:latest
+
+openshift_cluster_content:
+- object: Environment Setup
+  content:
+  - name: Create Projects
+    template: "https://raw.githubusercontent.com/redhat-cop/cluster-lifecycle/v3.9.0/files/projectrequest/template.yml"
```

Comment:
Can we switch to use `{{ openshift_templates_raw }}/{{ openshift_templates_raw_version }}/project-requests/create-project.yml`?